frontend/packages/dev-console/console-extensions.json

```
@@ -56,6 +56,35 @@
       ]
     }
   },
+  {
+    "type": "console.cluster-configuration/item",
+    "properties": {
+      "id": "dev-console.PinnedResourcesConfiguration",
+      "groupId": "developer",
+      "label": "%devconsole~Pre-pinned navigation items%",
+      "description": "%devconsole~A list of default resouces saved.%",
```

typo, and then I would recommend to help the cluster-admin what does this mean exactly:

```suggestion
      "description": "%devconsole~Resource links that are shown below the standard navigation items. These pinned resources can still be changed by the user.%",
```